src/redux/signup/signupReducer.js

```diff
@@ -19,7 +19,7 @@ const signupReducer = (state = initialState, action) => {
         ...state,
         user: action.payload.user,
         isFetching: false,
-        message: action.payload.message
+        message: action.payload.success
```

Comment:
Switch to message instead, the boolean is probably not something that is helpful to show for the user on success!

Reply:
Agree, fixed it with my latest commit.